nautobot/core/settings_funcs.py

```diff
@@ -82,3 +84,84 @@ def parse_redis_connection(redis_database):
         return f"{redis_scheme}://{redis_creds}{redis_host}?db={redis_database}"
     else:
         return f"{redis_scheme}://{redis_creds}{redis_host}:{redis_port}/{redis_database}"
+
+
+def setup_structlog_logging(
+    django_logging: dict,
+    django_apps: list,
+    django_middleware: list,
+    log_level="INFO",
+    root_level="INFO",
+    debug=False,
+    debug_db=False,
+    plain_format=False,
+) -> None:
+    """Set up structlog logging for Django."""
+    django_logging["version"] = 1
+    django_logging["disable_existing_loggers"] = True
+    if "test" in sys.argv:
+        django_logging["handlers"] = {
+            "null_handler": {
+                "level": "INFO",
```

Comment:
Not `log_level`?

Reply:
`level` is the correct key here.

Reply:
sorry, I meant instead of `"INFO"` being hard-coded. Same comment below.

Reply:
For testing, it was hardcoded to `INFO` in the original settings, so this keeps the same behavior.

As it is using `"class": "logging.NullHandler"`, it disables log anyway.